doc/cephfs/fs-volumes.rst

```diff
@@ -97,6 +97,43 @@ The CephX IDs authorized to <vol_name> need to be reauthorized to <new_vol_name>
 on-going operations of the clients using these IDs may be disrupted. Mirroring is
 expected to be disabled on the volume.
 
+Fetch the information of a CephFS volume using::
+
+    $ ceph fs volume info <vol_name>
```

Comment:
I think this is example so ```vol_name``` should be there instead of ```<vol_name>``` (angular brackets not needed)